editor/editor_inspector.cpp

```diff
@@ -919,7 +919,6 @@ void EditorProperty::menu_option(int p_option) {
 		} break;
 		case MENU_OPEN_DOCUMENTATION: {
 			ScriptEditor::get_singleton()->goto_help(doc_path);
-			EditorNode::get_singleton()->set_visible_editor(EditorNode::EDITOR_SCRIPT);
```

Comment:
`ScriptEditor::get_singleton()->goto_help()` already calls this function, so this line was redundant.